vision/beta/cloud-client/src/main/java/com/example/vision/Detect.java

```diff
@@ -101,7 +103,8 @@ public static void argsHelper(String[] args, PrintStream out) throws Exception,
               + "\n"
               + "Commands:\n"
               + "\tfaces | labels | landmarks | logos | text | safe-search | properties"
-              + "| web | web-entities | web-entities-include-geo | crop | ocr \n"
+              + "| web | web-entities | web-entities-include-geo | crop | ocr | object-localizer\n"
```

Comment:
Let's switch `object-localizer` to `object-localization` and for all the following occurrences.